atomdb/datasets/gaussian/__init__.py

```diff
@@ -96,6 +121,60 @@ def eval_orbs_density(one_density_matrix, orb_eval):
     return density
 
 
+def eval_radial_d_density(one_density_matrix, basis, points):
+    """Compute the radial derivative of the density.
+
+    For a set of points, compute the radial derivative of the density
+    given the one-electron density matrix and the basis set.
+
+    Parameters
+    ----------
+    one_density_matrix : np.ndarray(K_orb, K_orb)
+        One-electron density matrix (1DM) from K orbitals
+    basis : gbasis.basis.Basis
+        Basis set used to evaluate the radial derivative of the density
+    points : np.ndarray(N, 3)
+        Set of points where the radial derivative of the density is evaluated
+
+    Returns
+    -------
+    radial_d_density : np.ndarray(N)
+        Radial derivative of the density at the set of points
+    """
+    rho_grad = evaluate_density_gradient(one_density_matrix, basis, points)
+    # compute unitary vectors for the directions of the points
+    unitvect_pts = points / np.linalg.norm(points, axis=1)[:, None]
+    # compute the radial derivative of the density
+    return np.einsum("ij,ij->i", unitvect_pts, rho_grad)
+
+
+def eval_radial_dd_density(one_density_matrix, basis, points):
+    """Compute the radial derivative of the density.
```

Comment:
This is a minor typo, here should say: radial **second** derivative, and also in L154

Reply:
Please feel free to make any changes you see fit